tests/interlocked_hl_ut/interlocked_hl_ut.c

```diff
@@ -867,4 +873,56 @@ TEST_FUNCTION(InterlockedHL_SetAndWakeAll_succeeds)
 }
 #endif
 
+/* InterlockedHL_DecrementAndWake */
+
+/*Tests_SRS_INTERLOCKED_HL_44_001: [ If address is NULL then InterlockedHL_DecrementAndWake shall fail and return INTERLOCKED_HL_ERROR. ]*/
+TEST_FUNCTION(InterlockedHL_DecrementAndWake_with_NULL_address_fails)
+{
+    // arrange
+    INTERLOCKED_HL_RESULT result;
+
+    // act
+    result = InterlockedHL_DecrementAndWake(NULL);
+
+    // assert
+    ASSERT_ARE_EQUAL(INTERLOCKED_HL_RESULT, INTERLOCKED_HL_ERROR, result);
+}
+
+TEST_FUNCTION(InterlockedHL_DecrementAndWake_with_Valid_Inputs_success)

```

Comment:
>InterlockedHL_DecrementAndWake_with_Valid_Inputs_success [](http://example.com/codeflow?start=14&length=56)

Seems like a missing Tests_SRS for this test

Reply:
Oh I see, you put them in ln. 912.

Reply:
modified and put the SRS in the function header